index.html

```diff
@@ -38,5 +38,106 @@ <h1>I'm Shimels, Glad to see you!</h1>
         <a href="#"> <img src="icones/Vector.png" alt="" /></a>
       </section>
     </main>
+
+    <main class="project-container">
+      <div class="card">
+        <img src="images/bk-image.jpg" alt="project image" />
+        <h2>Tonic</h2>
+        <h4>
+          <span>CANOPY</span> &nbsp; ● &nbsp; Back End Dev &nbsp; ● &nbsp; 2015
+        </h4>
```

Comment:
- [ ] Nice work implementing the project information :clap: However it would be nice to align them on the same line to follow the Figma design.
### Your design :point_down: 
![Screenshot from 2023-07-07 16-57-55](https://github.com/shimels1/portfolio_microverse/assets/91142432/6e7bbaa3-a5fe-4d92-9571-591ec8020105)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-07-07 16-58-45](https://github.com/shimels1/portfolio_microverse/assets/91142432/e038354e-8b05-4f5a-b494-bf21d55b0115)